libs/design-system/data-viewer/src/lib/types/data-viewer.schema.ts

```diff
@@ -0,0 +1,37 @@
+import { z } from 'zod';
+
+/** Type for viewer card data */
+export type ViewerCardData = z.infer<typeof ViewerCardDataSchema>;
+/** Schema for viewer card data */
+export const ViewerCardDataSchema = z.object({
+  name: z.string(),
+  metadata: z.string().url(),
+  ai: z.string().url().optional(),
+  png: z.string().url().optional(),
+  svg: z.string().url().optional(),
+  crosswalk: z.string().url().optional(),
+  threeDimImage: z.string().url().optional(),
+  alt: z.string().optional(),
+});
+
+/** All viewer card data for an organ */
+export type OrganData = z.infer<typeof OrganDataSchema>;
+/** Schema for organ data */
+export const OrganDataSchema = z.object({
+  name: z.string(),
+  icon: z.string().url(),
+  viewerCardData: ViewerCardDataSchema.array(),
+});
+
+/** Organ version data including organ data */
+export type OrganVersionData = z.infer<typeof OrganVersionDataSchema>;
+/** Schema for organ version data */
+export const OrganVersionDataSchema = z.object({
```

Comment:
Rename and update to
```ts
const ViewerVersionDataSchema = z.object({
  version: z.string(),
  label: z.string(),
  crosswalkUrl: z.string().url().optional(),
  extractionsSitesUrl: z.string().url().optional(),
  referenceOrgansUrl: z.string().url().optional(),
  organData: ViewerOrganDataSchema.array()
})
```

Reply:
I'm renaming to ReleaseVersionDataSchema which I think describes it better